pkg/loop/internal/core/services/capability/capabilities_test.go

```diff
@@ -240,7 +240,13 @@ func Test_Capabilities(t *testing.T) {
 		err = client.Close()
 		require.NoError(t, err)
 
-		_, isOpen := <-ch
+		// Closing the client will result in an error being
+		// bubbled back to the client.
+		resp := <-ch
+		assert.NotNil(t, resp.Err)
```

Comment:
i'd prefer to be using typed errors and asserting that we get the expected error.